docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -143,34 +144,34 @@ Be sure to observe and follow team conventions for how PRs should be named and d
 
 **Title:** [module_name] Quick description   
 
-**[Contents](../../../.github/PULL_REQUEST_TEMPLATE.md) of initial comment:** Should include a brief summary of the changes and rationale, links to related issues or PRs, and testing instructions. 
+**[Contents](../../../.github/PULL_REQUEST_TEMPLATE.md) of the initial comment:** Include a brief summary of the changes and rationale, links to related issues or PRs, and testing instructions if there are any. 
 
-**Linking issues:** Use GitHub keywords like `Fixes` or `Resolves` so that the issue is closed automatically when the PR is merged.
-**Note** that automatic issue-linking and issue-closing only works on the default branch.  If the PR is not on the default branch, you’ll need to manually close the issue after merge. 
+**Linking issues:** Use GitHub keywords like `Fixes` or `Resolves` so that related issues are closed automatically when the PR is merged. Do not use these key words if you are only addressing one point raised in an issue. This will close the entire issue and leave unresolved bugs.
```

Comment:
```suggestion
**Linking issues:** Use GitHub keywords like `Fixes` or `Resolves` so that related issues are closed automatically when the PR is merged. Do not use these keywords if you are only addressing one point raised in an issue. This will close the entire issue and leave unresolved bugs in the codebase.
```